src/theme/components/accordion.ts

```diff
@@ -1,21 +1,28 @@
 const containerStyles = {
   borderLeftWidth: "1px",
   borderRightWidth: "1px",
-  borderColor: "ui.gray.medium",
   width: "100%",
+  _hover: {
+    borderLeftWidth: "1px",
+    borderRightWidth: "1px",
+    borderColor: "ui.gray.dark",
+    // The bottom border of the accordion item being hovered
+    // over is hidden beneath the top border of the following
+    // sibling, so we need to grab and change that border, as well.
+    "& + .chakra-accordion__item": {
+      borderTopColor: "ui.gray.dark",
```

Comment:
This should also be `ui.gray.medium`.

Reply:
Same comment as above.

Reply:
Ditto for me.  Same diffident comment as above.

Reply:
No worries!! Thanks for clarifying.